docs/index.rst

```diff
@@ -46,51 +53,38 @@ that have some or all of the following requirements:
     Girder also includes everything needed for pluggable user management and
     authentication out of the box and adheres to best practices in web security.
     The system can be configured to securely store credentials itself, or defer
-    to third-party authentication services such as OAuth or LDAP.
+    to third-party authentication services such as OAuth or LDAP/Active Directory.
 
 * **Authorization management**
     Girder supports a simple access control scheme that allows both user-based
     and role-based access control on resources managed in the system. The project
     has undergone rigorous security audits and has extensive automated testing
     to exercise authorization behavior and ensure correctness.
 
-For an overview of the concepts present in Girder, we recommend checking out the :doc:`user-guide`.
+* **Offline analytics / data processing**
```

Comment:
Howsabout "asynchronous remote execution"?

the title section could be "Asynchronous remote analytics / data processing" and the paragraph text could "also deploy its companion offline processing tool, GW, which provides asynchronous remote execution."

Reply:
"Offline" almost certainly is the wrong word. I'd go with "offloaded" or "delegated" to mean that the computation gets done on a dedicated process separate from Girder itself. "Asynchronous" is close but because we're in web-world, the term feels overloaded.

Reply:
What about "Distributed analytics/execution/data processing " ?